src/core/symbology/qgssymbollayerutils.h

```diff
@@ -897,6 +897,14 @@ class CORE_EXPORT QgsSymbolLayerUtils
      */
     static QgsStringMap evaluatePropertiesMap( const QMap<QString, QgsProperty> &propertiesMap, const QgsExpressionContext &context );
 
+    /**
+     * Calculate the minimum size in pixels of a symbol tile given the symbol \a width and \a height and the grid rotation \a angle in radians.
+     * The method makes approximations and can modify \a angle in order to generate the smallest possible tile.
+     * \note Angle must be >= 0 and < 2 * PI
+     * \since QGIS 3.30
```

Comment:
If you add \returns and \param arguments here, then sipify will correctly handle the INOUT argument in the docstring.